egi2bids/commands/egi2bids.py

```diff
@@ -12,35 +12,39 @@ def run():
     parser.add_argument(
         "mff_source",
         type=str,
-        help="path to the .mff file.",
+        help=(
+            "path to the input mff file. "
+            "Can be a '.mmf' folder or a '.mff.tar', "
+            "'.mff.tar.gz' or '.mff.zip' archive."
+        ),
     )
     parser.add_argument(
         "bids_root",
         type=str,
         help="path to the BIDS root.",
     )
     parser.add_argument(
-        "-s",
+        "-sub",
         "--subject",
         type=str,
         metavar="str",
         help="Subject ID.",
         required=True,
     )
     parser.add_argument(
-        "-t",
-        "--task",
+        "-ses",
+        "--session",
         type=str,
         metavar="str",
-        help="Task.",
```

Comment:
Shouldn't this be `type=int` and `metavar="int"`?

Reply:
from the BIDS specification [here](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#filesystem-structure) session is a `string`:
```If data for the subject were acquired across multiple sessions, then within the subject directory resides subdirectories named "ses-<label>", where string "<label>" is substituted with a unique identification label for each session.```